[25/Apr/2020 12:35:32] ERROR [django.request:228] Internal Server Error: /api/code-gen/generate
[25/Apr/2020 12:35:32] ERROR [django.server:154] "POST /api/code-gen/generate?table_name=tDOCTORS HTTP/1.1" 500 162
[27/Apr/2020 00:57:13] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 00:57:13] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=&password= HTTP/1.1" 500 125
[27/Apr/2020 00:57:31] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 00:57:31] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=&password= HTTP/1.1" 500 125
[27/Apr/2020 00:59:00] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 00:59:00] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:00:44] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:00:44] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:01:48] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:01:48] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:04:47] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:04:47] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:05:33] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:05:33] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:06:49] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:06:49] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:08:27] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:08:27] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:08:27] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 59
[27/Apr/2020 01:11:05] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:11:05] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:14:19] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:14:19] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:14:34] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:14:34] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:15:02] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:15:02] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:22:30] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:22:30] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:29:55] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:29:55] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 131
[27/Apr/2020 01:32:12] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:32:12] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:39:08] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:39:08] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 01:39:37] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:39:37] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 155
[27/Apr/2020 01:44:38] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:44:38] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 01:45:43] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:45:43] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 01:46:27] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:46:27] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 01:46:54] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:46:54] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 01:48:20] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:48:20] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 01:48:41] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 01:48:41] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 19:18:31] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 19:18:31] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 124
[27/Apr/2020 19:19:00] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 19:19:00] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
[27/Apr/2020 19:20:21] ERROR [django.request:228] Internal Server Error: /api/authentication/authenticate
[27/Apr/2020 19:20:21] ERROR [django.server:154] "POST /api/authentication/authenticate?mobile_number=9611798835&password=12345 HTTP/1.1" 500 125
